lib/Echidna/Campaign.hs

```diff
@@ -46,6 +54,7 @@ data CampaignConf = CampaignConf { testLimit     :: Int
                                  , knownCoverage :: Maybe (Map W256 (Set Int))
                                    -- ^ If applicable, initially known coverage. If this is 'Nothing',
                                    -- Echidna won't collect coverage information (and will go faster)
+                                 , seed          :: Maybe Int
```

Comment:
Please add a comment documenting

Reply:
Also, I don't think this should be a `Maybe Int`. I think that I'd prefer it just be a StdGen that we build intelligently at parse time. This seems like a logical mismatch as-is